### DISCUSSION TITLE
[NOVA TRILHA] HTML5

### DISCUSSION THREAD

**Discussion by LuizFernando-TC** [codaqui/institucional#70] [Trilhas de Ensino]
Issue: https://github.com/codaqui/institucional/issues/23
Vamos adicionar uma nova trilha na Codaqui?
Sugiro termos uma trilha de HTML5, uma linguagem de marcação muito importante na rede web para formatar páginas.

**Comment by LuizFernando-TC** [comment]
Issue: https://github.com/codaqui/institucional/issues/23

**Comment by endersonmenezes** [comment]
Trilha aprovada e chamando colaboradores para discutir tecnicamente na Issue.
